orthoinference/src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -290,61 +271,60 @@ public static void readENSGMappingFile(String toSpecies, String pathToOrthopairs
 			String[] tabSplit = currentLine.split("\t");
 			String ensgKey = tabSplit[0];
 			String[] spaceSplit = tabSplit[1].split(" ");
-			for (String proteinId : spaceSplit) 
+			for (String proteinId : spaceSplit)
 			{
-				String[] colonSplit = proteinId.split(":");
-				if (ensgMappings.get(colonSplit[1]) == null)
+				if (ensgMappings.get(proteinId) == null)
 				{
 					List<String> singleArray = new ArrayList<String>();
 					singleArray.add(ensgKey);
-					ensgMappings.put(colonSplit[1], (ArrayList<String>) singleArray);
+					ensgMappings.put(proteinId, (ArrayList<String>) singleArray);
```

Comment:
For putting an array of length 1 into a map, you can do:
`ensgMappings.put(proteinId, Collections.singleton(ensgKey));`

`ensgMappings` can also have its declaration at the start of the class as `Map<String, List<String>> ensgMappings = new HashMap<>();` so a cast to `ArrayList<String>` isn't needed.